setup-scripts/install.sh

```diff
@@ -126,9 +121,41 @@ if [ -d "broker" ]; then
 else
   git clone -b "$BROKER_VERSION" --single-branch --depth 1 https://github.com/sparkswap/broker.git
   cd broker
-  npm run build -- -e=$IP_ADDRESS
+
+  # Build images locally for the broker
+  if [[ $BUILD == "true" ]]; then
+    npm run build -- -e=$IP_ADDRESS
+  else
+    npm run build -- -e=$IP_ADDRESS --no-docker
+  fi
+fi
+
+# Detect LND-Engine version
+msg "Detecting LND Version from the Broker" $WHITE
+LND_ENGINE_VERSION=$(echo "console.log($(npm list lnd-engine --json).dependencies['lnd-engine'].version);" | node)
```

Comment:
Since we already have node installed here, we can grab the version from npm (from the broker)

Reply:
won't this only work if `BUILD` is true and we have cloned the broker?

Reply:
@treygriffith only build is true, no matter what we have to clone the broker (for the docker-compose files)